internal/impl/io/input_websocket.go

```diff
@@ -176,9 +178,27 @@ func (w *websocketReader) Connect(ctx context.Context) error {
 	}
 
 	if len(w.openMsg) > 0 {
-		if err := client.WriteMessage(openMsgType, w.openMsg); err != nil {
-			return err
+		if openMsgType == websocket.BinaryMessage {
```

Comment:
This is weird. Why not simply add a config calls `open_messages` which is a list of strings? The meansings of 'binary' and 'text' are very confusing. `open_messages` makes it clear that it's a list of messages to be sent, not just one.

Reply:
Just keep backward compatible, I wont change the binary process here

Reply:
change the config will involve more test and UI change, so just keep a simple change for now
we can consider update that later